scripts/masterCron.ts

```diff
@@ -81,12 +84,27 @@ async function settleAuctions(
         gasPrice: newGasPrice,
         gasLimit: gasLimits["settleAuction"],
       });
+
       await log(`GnosisAuction-settleAuction()-${auctionID}: ${tx.hash}`);
     } catch (error) {
       await log(
         `@everyone GnosisAuction-settleAuction()-${auctionID}: failed with error ${error}`
       );
     }
+
+    try {
```

Comment:
better to separate _claimFromParticipantOrder_ into another method?

Reply:
Looks fine how we're doing it now